_data/simple-icons.json

```diff
@@ -9896,6 +9901,11 @@
             "hex": "3F6184",
             "source": "https://github.com/wallabag/logo/blob/f670395da2d85c3bbcb8dcfa8d2a339d8af5abb0/_default/icon/svg/logo-icon-black-no-bg.svg"
         },
+        {
+            "title": "Walmart",
+            "hex": "0071CE",
+            "source": "https://cdn.corporate.walmart.com/12/9d/06ecf3584e91acccc37162f09714/logowhite.svg"
+        },
```

Comment:
Can you revert walmart changes from this PR?

Reply:
ups. Yes, of course